internal/metrics/types.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+const (
+	MetricTypeGauge   = "gauge"
+	MetricTypeCounter = "counter"
+
+	ErrorParseMetricValue  = "error parse metric value"
```

Comment:
1. Если объявляешь ошибку таким образом, то это более идиоматично делать так:

```suggestion
	ErrParseMetricValue  = errors.New("error parse metric value")
```

2. А зачем тебе вообще определять эту ошибку ? Она нигде не проверяется, зачем ее "типизировать" вместо простого `fmt.Errorf` ?

Reply:
1. +
2. Согласен, тоже +